app/src/main/java/com/example/myapplication/PitFragment.java

```diff
@@ -106,8 +106,15 @@ public void onClick(View view) {
                 NavHostFragment.findNavController(PitFragment.this)
                         .navigate(R.id.action_pitFragment_to_ThirdFragment);
             }
+<<<<<<< HEAD:app/src/main/java/com/example/myapplication/PitFragment.java
```

Comment:
It looks like there is or was a merge conflict here that wasn't fully cleaned out

Reply:
i fixed this